### PR TITLE
differences.md fixes

### PR DESCRIPTION
**Pull request by danny-goldberg-rally**

### CODE REVIEW

File: differences.md [hunk 1]
```diff
@@ -18,11 +18,11 @@ play-json always writes a `"key":"value"` pair regardless of whether it's the ca
 
 weepickle works [differently](http://www.lihaoyi.com/upickle/#Defaults). If a field is missing upon deserialization, weepickle uses the default value if one exists. If a field at serialization time has the same value as the default, weepickle leaves it out of the serialized blob.
 
-OpenAPI 3 [has support](https://swagger.io/specification/#schemaObject) for weepickle's behavior:
```
Comment (danny-goldberg-rally) [inline differences.md:21]: this made it sound like play-json doesn't have support for weepickle's default behavior

Comment (htmldoug) [reply differences.md:21]: It doesn't, does it? You'd have to drop down to custom `Writes` to achieve what weePickle can do with the macros alone.

**Review by htmldoug — APPROVED** [review]
thanks!

File: differences.md (hunk 1 above)
Comment (danny-goldberg-rally) [reply differences.md:21]: play-json does support `weePickle`'s default behavior, but doesn't support `uPickle`'s default behavior

**Review by russellremple — APPROVED** [review]
lgtm

**Pull request merged**